### PR TITLE
localStorage

### PR DESCRIPTION
**Pull request by ninapeters**
Hi there! I've added a localStorage to my app.
In addition, I discovered a small typo in this step that I corrected. Unfortunately, a few more files are displayed as changed because of this.

I look forward to every review and comment. Thank you in advance!

### CODE REVIEW

File: src/lib/localStorage.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  try {
+    return JSON.parse(localStorage.getItem(key))
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+export function saveToLocal(key, value) {
+  localStorage.setItem(key, JSON.stringify(value))
+}
```
Comment (FabianAlleblas) [inline src/lib/localStorage.js:11]: I'm missing the tests?

**Review by FabianAlleblas — COMMENTED** [review]
Nice one 🙌

**Review by JulianBlohm — COMMENTED** [review]
Great work, there is nothing better than a simple local storage PR.
You don't need error handling for local storage, like it is now. 
I am a bit confused about your file structure, would say there is no need to have the pages in the component/page folder. Why not having them just like the App.js in src folder? 
Out of that you don't need styleguidist for pages, just for your components.

File: src/lib/localStorage.js (hunk 1 above)
Comment (ninapeters) [reply src/lib/localStorage.js:11]: 🐸

File: src/hooks/useArtist.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { loadFromLocal, saveToLocal } from '../lib/localStorage'
+
+export default function useArtist({ artistData }) {
+  const [savedSongsList, setSavedSongsList] = useState(
+    loadFromLocal('SavedSongsList') ?? []
+  )
+
+  useEffect(() => {
+    saveToLocal('SavedSongsList', savedSongsList)
```
Comment (Hecktors) [inline src/hooks/useArtist.js:10]: I guess starting with lowercase letter would be more conventional

File: src/hooks/useArtist.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { loadFromLocal, saveToLocal } from '../lib/localStorage'
+
+export default function useArtist({ artistData }) {
```
Comment (Hecktors) [inline src/hooks/useArtist.js:4]: What about using `propTypes` like you made in the other js files?

File: src/components/Pages/ArtistPage/ArtistPage.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ export default function ArtistPage({
             <Button
               id={id}
               handleClick={() => saveSong(id)}
-              disabled={SavedSongsList?.some((song) => song.id === id)}
+              disabled={savedSongsList?.some((song) => song.id === id)}
```
Comment (Hecktors) [inline src/components/Pages/ArtistPage/ArtistPage.js:40]: I would name it `isDisabled` to make the boolean type clear.

**Review by Hecktors — COMMENTED** [review]
Nice job. Love your test approach!

**Pull request merged**